poprox-db/tests/test_create_scrape_table.py

```diff
@@ -0,0 +1,59 @@
+import psycopg2
+from sqlalchemy import create_engine
+
+# Database connection parameters
+conn_params = {
```

Comment:
This is not how the rest of the codebase is connecting -- so we'll need to clean this part up before merging.